### PR TITLE
[FAB-17662] Remove Get prefix from exported ConfigTX functions

### PR DESCRIPTION
**Pull request by stephyee**
#### Type of change

- Improvement (improvement to code, performance, etc)

#### Description

Remove Get prefix from exported ConfigTX functions. 

#### Related issues
[FAB-17662](https://jira.hyperledger.org/browse/FAB-17662)

### CODE REVIEW

File: pkg/config/application.go [hunk 1]
```diff
@@ -24,36 +24,36 @@ type Application struct {
...
 	}
 
-	capabilities, err := c.GetApplicationCapabilities()
+	capabilities, err := c.ApplicationCapabilities()
 	if err != nil {
 		return Application{}, fmt.Errorf("retrieving application capabilities: %v", err)
 	}
 
-	policies, err := c.GetPoliciesForApplication()
+	policies, err := c.PoliciesForApplication()
```
Comment (caod123) [inline pkg/config/application.go]: We may want to consider renaming this to `ApplicationPolicies` for consistency with the other retrievers.

File: pkg/config/orderer.go [hunk 2]
```diff
@@ -203,7 +203,7 @@ func (c *ConfigTx) GetOrdererConfiguration() (Orderer, error) {
 	}
 
 	// POLICIES
-	policies, err := c.GetPoliciesForOrderer()
+	policies, err := c.PoliciesForOrderer()
```
Comment (caod123) [inline pkg/config/orderer.go]: Likewise `OrdererPolicies`

File: pkg/config/policies.go [hunk 3]
```diff
@@ -43,8 +43,8 @@ func (c *ConfigTx) UpdateConsortiumChannelCreationPolicy(consortiumName string,
 	return nil
 }
 
-// GetPoliciesForConsortiums returns a map of policies for channel consortiums.
-func (c *ConfigTx) GetPoliciesForConsortiums() (map[string]Policy, error) {
+// PoliciesForConsortiums returns a map of policies for channel consortiums.
+func (c *ConfigTx) PoliciesForConsortiums() (map[string]Policy, error) {
```
Comment (caod123) [inline pkg/config/policies.go]: `ConsortiumsPolicies`

File: pkg/config/policies.go [hunk 4]
```diff
@@ -53,8 +53,8 @@ func (c *ConfigTx) GetPoliciesForConsortiums() (map[string]Policy, error) {
 	return getPolicies(consortiums.Policies)
 }
 
-// GetPoliciesForConsortium returns a map of policies for a specific consortium.
-func (c *ConfigTx) GetPoliciesForConsortium(consortiumName string) (map[string]Policy, error) {
+// PoliciesForConsortium returns a map of policies for a specific consortium.
+func (c *ConfigTx) PoliciesForConsortium(consortiumName string) (map[string]Policy, error) {
```
Comment (caod123) [inline pkg/config/policies.go]: `ConsortiumPolicy`

File: pkg/config/policies.go [hunk 5]
```diff
@@ -63,8 +63,8 @@ func (c *ConfigTx) GetPoliciesForConsortium(consortiumName string) (map[string]P
 	return getPolicies(consortium.Policies)
 }
 
-// GetPoliciesForConsortiumOrg returns a map of policies for a specific consortium org.
-func (c *ConfigTx) GetPoliciesForConsortiumOrg(consortiumName, orgName string) (map[string]Policy, error) {
+// PoliciesForConsortiumOrg returns a map of policies for a specific consortium org.
+func (c *ConfigTx) PoliciesForConsortiumOrg(consortiumName, orgName string) (map[string]Policy, error) {
```
Comment (caod123) [inline pkg/config/policies.go]: `ConsortiumOrgPolicies`

File: pkg/config/policies.go [hunk 6]
```diff
@@ -73,8 +73,8 @@ func (c *ConfigTx) GetPoliciesForConsortiumOrg(consortiumName, orgName string) (
 	return getPolicies(org.Policies)
 }
 
-// GetPoliciesForOrderer returns a map of policies for channel orderer.
-func (c *ConfigTx) GetPoliciesForOrderer() (map[string]Policy, error) {
+// PoliciesForOrderer returns a map of policies for channel orderer.
```
Comment (caod123) [inline pkg/config/policies.go]: `OrdererPolicies`

File: pkg/config/policies.go [hunk 7]
```diff
@@ -83,8 +83,8 @@ func (c *ConfigTx) GetPoliciesForOrderer() (map[string]Policy, error) {
 	return getPolicies(orderer.Policies)
 }
 
-// GetPoliciesForOrdererOrg returns a map of policies for a specific org.
-func (c *ConfigTx) GetPoliciesForOrdererOrg(orgName string) (map[string]Policy, error) {
+// PoliciesForOrdererOrg returns a map of policies for a specific org.
+func (c *ConfigTx) PoliciesForOrdererOrg(orgName string) (map[string]Policy, error) {
```
Comment (caod123) [inline pkg/config/policies.go]: `OrdererOrgPolicies`

File: pkg/config/policies.go [hunk 8]
```diff
@@ -93,8 +93,8 @@ func (c *ConfigTx) GetPoliciesForOrdererOrg(orgName string) (map[string]Policy,
 	return getPolicies(org.Policies)
 }
 
-// GetPoliciesForApplication returns a map of policies for application config group.
-func (c *ConfigTx) GetPoliciesForApplication() (map[string]Policy, error) {
+// PoliciesForApplication returns a map of policies for application config group.
+func (c *ConfigTx) PoliciesForApplication() (map[string]Policy, error) {
```
Comment (caod123) [inline pkg/config/policies.go]: See above

File: pkg/config/policies.go [hunk 9]
```diff
@@ -103,9 +103,9 @@ func (c *ConfigTx) GetPoliciesForApplication() (map[string]Policy, error) {
 	return getPolicies(application.Policies)
 }
 
-// GetPoliciesForApplicationOrg returns a map of policies for specific application
+// PoliciesForApplicationOrg returns a map of policies for specific application
```
Comment (caod123) [inline pkg/config/policies.go]: See above

File: pkg/config/policies.go [hunk 10]
```diff
@@ -114,8 +114,8 @@ func (c *ConfigTx) GetPoliciesForApplicationOrg(orgName string) (map[string]Poli
 	return getPolicies(orgGroup.Policies)
 }
 
-// GetPoliciesForChannel returns a map of policies for channel configuration.
-func (c *ConfigTx) GetPoliciesForChannel() (map[string]Policy, error) {
+// PoliciesForChannel returns a map of policies for channel configuration.
```
Comment (caod123) [inline pkg/config/policies.go]: See above

**Review by caod123 — CHANGES_REQUESTED** [review]
Some comments about consistency of the Policy retrieval function names

File: pkg/config/application_test.go [hunk 11]
```diff
@@ -602,7 +602,7 @@ func TestGetAnchorPeer(t *testing.T) {
 	err = c.AddAnchorPeer("Org1", expectedAnchorPeer)
 	gt.Expect(err).NotTo(HaveOccurred())
 
-	anchorPeers, err := c.GetAnchorPeers("Org1")
+	anchorPeers, err := c.AnchorPeers("Org1")
```
Comment (DereckLuo) [inline pkg/config/application_test.go:605]: function name probably should change to `TestAnchorPeers`

File: pkg/config/application_test.go [hunk 12]
```diff
@@ -657,7 +657,7 @@ func TestGetAnchorPeerFailures(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			t.Parallel()
 			gt := NewGomegaWithT(t)
-			_, err := c.GetAnchorPeers(test.orgName)
+			_, err := c.AnchorPeers(test.orgName)
```
Comment (DereckLuo) [inline pkg/config/application_test.go:660]: function name probably should change to `TestAnchorPeerFailures`

File: pkg/config/organization.go [hunk 13]
```diff
@@ -25,8 +25,8 @@ func (c *ConfigTx) GetApplicationOrg(orgName string) (Organization, error) {
 	return getOrganization(orgGroup, orgName)
 }
 
-// GetOrdererOrg retrieves an existing org from an orderer organization config group.
-func (c *ConfigTx) GetOrdererOrg(orgName string) (Organization, error) {
+// OrdererOrg retrieves an existing org from an orderer organization config group.
```
Comment (DereckLuo) [inline pkg/config/organization.go:28]: missed `GetConsortiumOrg` in this file

File: pkg/config/organization_test.go [hunk 14]
```diff
@@ -30,7 +30,7 @@ func TestGetOrganization(t *testing.T) {
 	gt.Expect(expectedOrg).To(Equal(org))
 }
 
-func TestGetApplicationOrg(t *testing.T) {
+func TestApplicationOrg(t *testing.T) {
```
Comment (DereckLuo) [inline pkg/config/organization_test.go:33]: missed `TestGetConsortiumOrg` in this file.

**Review by DereckLuo — COMMENTED** [review]
LGTM. just missed out couple minor places for `Get` prefix.

**Comment by caod123** [comment]
@stephyee can you rebase to pick up the changes from FAB-17665 and FAB-17660

File: pkg/config/policies.go [hunk 15]
```diff
@@ -53,8 +53,8 @@ func (c *ConfigTx) GetPoliciesForConsortiums() (map[string]Policy, error) {
 	return getPolicies(consortiums.Policies)
 }
 
-// GetPoliciesForConsortium returns a map of policies for a specific consortium.
-func (c *ConfigTx) GetPoliciesForConsortium(consortiumName string) (map[string]Policy, error) {
+// ConsortiumPolicies returns a map of policies for a specific consortium.
+func (c *ConfigTx) ConsortiumPolicies(consortiumName string) (map[string]Policy, error) {
```
Comment (caod123) [inline pkg/config/policies.go]: This and ConsortiumsPolicies can be removed now that FAB-17665 is merged

File: pkg/config/policies.go [hunk 16]
```diff
@@ -43,8 +43,8 @@ func (c *ConfigTx) UpdateConsortiumChannelCreationPolicy(consortiumName string,
 	return nil
 }
 
-// GetPoliciesForConsortiums returns a map of policies for channel consortiums.
-func (c *ConfigTx) GetPoliciesForConsortiums() (map[string]Policy, error) {
+// ConsortiumsPolicies returns a map of policies for channel consortiums.
+func (c *ConfigTx) ConsortiumsPolicies() (map[string]Policy, error) {
```
Comment (caod123) [inline pkg/config/policies.go]: This can be removed now that FAB-17665 is merged

**Review by caod123 — CHANGES_REQUESTED** [review]

File: pkg/config/policies.go (hunk 15 above)
Comment (stephyee) [reply pkg/config/policies.go]: 🛠

File: pkg/config/policies.go (hunk 16 above)
Comment (stephyee) [reply pkg/config/policies.go]: 🛠

**Comment by stephyee** [comment]
@caod123 rebased

**Review by caod123 — APPROVED** [review]

**Pull request merged**